### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by peterstarling**
Updates all dependencies of root and each package/app to their latest minor version as per https://trello.com/c/NGLuNRP4/819-update-asap-server-dependencies

## Notes
Contains minor code changes
Formatting changed due to prettier upgrade

### CODE REVIEW

File: apps/auth0-rules/package.json [hunk 1]
```diff
@@ -10,17 +10,17 @@
     "watch:babel": "../../scripts/build-babel.sh watch"
   },
   "dependencies": {
```
Comment (jeysal) [inline apps/auth0-rules/package.json:12]: Gotta remember to push any changes to the built rules to auth0 (manual process) and check if everything works.

**Review by jeysal — COMMENTED** [review]
Yeah there are various issues popping up, I can't get this green by the end of today, will see tmrw. BTW if there were any changes made other than pure updates and prettier reformat that might be of interest lmk.

**Comment by jeysal** [comment]
Let's see if this passes and the deployment looks good

File: apps/auth0-rules/package.json (hunk 1 above)
Comment (jeysal) [reply apps/auth0-rules/package.json:12]: (done this already)

**Comment by jeysal** [comment]
image
when you update stuff and it makes your bundle bigger :(

**Comment by jeysal** [comment]
I'll LGTM once I've actually clicked through the live environment :D

**Comment by peterstarling** [comment]
> coverage
> free coverage
> yay

i dont even know why but whatever

**Review by jeysal — APPROVED** [review]
Env seems to work

**Pull request merged**